src/item.cpp

```diff
@@ -8086,6 +8086,10 @@ bool item::detonate( const tripoint &p, std::vector<item> &drops )
             // If it has a special effect just trigger it.
             apply_ammo_effects( p, ammo_type.ammo_effects );
         }
+        if( ammo_type.cookoff ) {
+            // If ammo type can burn, then create an explosion proportional to quantity.
+            explosion_handler::explosion( p, 3.0f*sqrtf(sqrtf(rounds_exploded / 25.0f)), 0.0f, false, 0 );
```

Comment:
```suggestion
            explosion_handler::explosion( p, 3.0f*sqrtf( sqrtf( rounds_exploded / 25.0f ) ), 0.0f, false, 0 );
```

Reply:
My bad - I knew about the rules for spacing but missed that one.

Reply:
There's an automated program called astyle that we use for code format. I suggest you use that, instead of doing it manually.

Reply:
I'll check it out